<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimeWwise</class>
 <widget class="QMainWindow" name="AnimeWwise">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnimeWwise</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>-1</y>
      <width>1091</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="configTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>1071</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainVLayout">
       <item>
        <layout class="QGridLayout" name="ioGrid">
         <item row="1" column="2">
          <widget class="QPushButton" name="changeAltInput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="changeInput">
           <property name="text">
            <string>Select</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="altInputPath">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="altInputLabel">
           <property name="text">
            <string>Diff folder (optional)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="inputLabel">
           <property name="text">
            <string>Input folder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="inputPath">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="separatorA">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="settingsGrid">
         <item row="0" column="1">
          <widget class="QComboBox" name="assetMap"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="assetMapLabel">
           <property name="text">
            <string>Asset map</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="separatorB">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadFilesButton">
         <property name="text">
          <string>Load file(s)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="browseTab">
     <attribute name="title">
      <string>Browse</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>611</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="extractTab">
     <attribute name="title">
      <string>Extract</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>1061</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainVLayout2">
       <item>
        <layout class="QHBoxLayout" name="outputLayout">
         <item>
          <widget class="QLabel" name="outputLabel">
           <property name="text">
            <string>Output folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeOutput">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputFormatLayout">
         <item>
          <widget class="QLabel" name="outputFormatLabel">
           <property name="text">
            <string>Output format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="outputFormat">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="separatorC">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="progressWrapperLayout">
         <item>
          <layout class="QHBoxLayout" name="totalProgressLayout">
           <item>
            <widget class="QLabel" name="totalProgressLabel">
             <property name="text">
              <string>Total progress</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="totalProgress">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="fileProgressLayout">
           <item>
            <widget class="QLabel" name="fileProgressLabel">
             <property name="text">
              <string>Per file progress</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="fileProgress">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="separatorD">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="extractLayout">
         <item>
          <widget class="QPushButton" name="extractAll">
           <property name="text">
            <string>Extract All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extractSelected">
           <property name="text">
            <string>Extract Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>1081</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>220</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionnot_working_here_yet">
   <property name="text">
    <string>not working here yet</string>
   </property>
  </action>
  <action name="actionExtractAll">
   <property name="text">
    <string>All files</string>
   </property>
  </action>
  <action name="actionExtractSelected">
   <property name="text">
    <string>Selected files</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inputPath</tabstop>
  <tabstop>changeInput</tabstop>
  <tabstop>altInputPath</tabstop>
  <tabstop>changeAltInput</tabstop>
  <tabstop>loadFilesButton</tabstop>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
